packages/dev/docs/pages/releases/2023-05-19.mdx

```diff
@@ -0,0 +1,81 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we are moving TagGroup and Column Resizing to GA. We also have some fixes to Calendars and have been busy making many improvements to the new React Aria Components we announced last release.
+date: 2023-05-19
```

Comment:
Just a reminder to update this date + header below + file name once we actually release

Reply:
yep, i set it to tomorrow, but it's possibly a little optimistic, haha